index.bs

```diff
@@ -546,7 +546,8 @@ interface WebTransport {
   /* a ReadableStream of WebTransportBidirectionalStream objects */
```

Comment:
One line above, we need the same argument.

Regarding names, we should probably have this type be a union of types.  When creating a unidirectional stream, you include "sending" options, but when creating a bidirectional stream, you might include both "sending" and "receiving" options.

Reply:
I wasn't planning on introducing this to bidirectional streams (the dictionary is named WebTransportUnidirectionalStreamOptions). Figuring out their semantics seems best deferred to when we have a use case for that, which seems out of scope for https://github.com/w3c/webtransport/issues/431.